x-pack/plugins/lens/public/data_views_service/loader.ts

```diff
@@ -47,6 +47,9 @@ export function convertDataViewIntoLensIndexPattern(
         meta: dataView.metaFields.includes(field.name),
         esTypes: field.esTypes,
         scripted: field.scripted,
+        isMapped: field.isMapped,
+        customLabel: field.customLabel,
+        runtimeField: field.runtimeField,
```

Comment:
Not sure if this is okay to extend. What was the reason to construct `IndexPatternField` instead of `DataViewField`?